sunpy/map/mapsequence.py

```diff
@@ -511,3 +511,24 @@ def all_meta(self):
         Return all the meta objects as a list.
         """
         return [m.meta for m in self.maps]
+
+    def save(self, filepath, filetype='auto', **kwargs):
+        """
+        Saves the maps in the map sequence to disk.
+        Each map from the sequence will be saved separately.
+
+        Currently this only supports saving files in the FITS format.
+
+        Parameters
+        ----------
+        filepath : str
+            Location to save file to.
+        filetype : str
+            'auto' or any supported file extension.
+        kwargs :
+            Any additional keyword arguments are passed to
+            `~sunpy.io.write_file`.
+        """
+        for index, map_seq in enumerate(self.maps):
+            _filepath = filepath + "_{}".format(index)
+            map_seq.save(_filepath, filetype, **kwargs)
```

Comment:
```suggestion
            map_seq.save(f"{filepath}_{index}", filetype, **kwargs)
```
But see my overarching comment